Add documentation on how NGINX Agent works and Install instructions (#74)
nginxinc/nginx-amplify-doc: content/faq/nginx-amplify-agent.md

   2552 ?        S      0:00 amplify-agent
   ```

For manual installation, please check the [user guide]({{< relref "/install-manage-amp-agent/installing-agent#installing-the-agent-manually" >}}).

### What Do I Need to Configure the NGINX Amplify Agent to Report Metrics Correctly?

> After you install and start the agent, it should start reporting right away, pushing aggregated data to the Amplify backend at regular 1-minute intervals. It'll take about a minute for the new system to appear in the Amplify web interface.

Reviewer: ```suggestion
Once you install NGINX Amplify Agent, it will automatically begin sending metrics. You can expect to see real-time metrics in the Amplify web interface within about a minute.
```

### What System Resources are Required?

> Under 10% of the CPU and a few dozen MBs of RSS memory will be consumed. If you notice any anomalies in the system resource consumption, please submit a support request through https://my.f5.com/, please attach the debug log to the support case.

Reviewer: ```suggestion
Expect to use under 10% of the CPU and a few dozen MBs of RSS memory. If you notice any unusual resource consumption, submit a support request at [MyF5](https://my.f5.com/) and attach the debug log to the case.
```

> We work hard to make the agent consume as little memory and CPU as possible. Usually, it should be under 10% of CPU usage and a few dozen MBs of RSS memory. If you notice any anomalies in the system resource consumption, please submit a support request through https://my.f5.com/, please attach the debug log to the support case.

Reviewer 2: Don't need the following: `We work hard to make the agent consume as little memory and CPU as possible.`

`Usually, it should be under 10% of CPU usage and a few dozen MBs of RSS memory.` -> `Under 10% of the CPU and a few dozen MBs of RSS memory will be consumed.`

> We work hard to make the agent consume as little memory and CPU as possible. Usually, it should be under 10% of CPU usage and a few dozen MBs of RSS memory. If you notice any anomalies in the system resource consumption, please submit a support request through https://my.f5.com/, please attach the debug log to the support case.

Reviewer 2: This is also in the FAQ - Do we need/want it in both of these pages?

### How Can I Override the User ID for the Agent to Use?

> Please check the [Configuring the Agent]({{< relref "/install-manage-amp-agent/configuring-agent#overriding-the-effective-user-id" >}}) section of the documentation.

Reviewer: ```suggestion
Refer to the [Configuring the Agent]({{< relref "/install-manage-amp-agent/configuring-agent#overriding-the-effective-user-id" >}}) section in the documentation.
```
